proto/greenfield/virtualgroup/query.proto

```diff
@@ -0,0 +1,104 @@
+syntax = "proto3";
+
+package greenfield.virtualgroup;
+
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "greenfield/virtualgroup/params.proto";
+import "greenfield/virtualgroup/types.proto";
+
+// this line is used by starport scaffolding # 1
+
+option go_package = "github.com/bnb-chain/greenfield/x/virtualgroup/types";
+
+// Query defines the gRPC query service.
+service Query {
+  // Parameters queries the parameters of the module.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/greenfield/virtualgroup/params";
+  }
+
+  // Queries a list of GlobalVirtualGroup items.
```

Comment:
The comment `a list of` is not correct. We may need to refine the comments in this file.

Reply:
Fixed.